include/Graphs/SVFGNode.h

```diff
@@ -90,18 +90,25 @@ class MRSVFGNode : public VFGNode
 class FormalINSVFGNode : public MRSVFGNode
 {
 private:
-    const MemSSA::ENTRYCHI* chi;
+    const MRVer* ver;
 
 public:
     /// Constructor
-    FormalINSVFGNode(NodeID id, const MemSSA::ENTRYCHI* entry): MRSVFGNode(id, FPIN), chi(entry)
+    FormalINSVFGNode(NodeID id, const MRVer* resVer): MRSVFGNode(id, FPIN)
     {
-        cpts = entry->getMR()->getPointsTo();
+        cpts = resVer->getMR()->getPointsTo();
+        ver = resVer;
     }
-    /// EntryCHI
-    inline const MemSSA::ENTRYCHI* getEntryChi() const
+
+    /// Return points-to of the MR
+    inline const PointsTo& getPointsTo() const
```

Comment:
I think this might be the last place before we can finally merge the patch.
Please remove `getPointsTo()` if the parent class already has this method.